## file 'GoToPoint.action'
# type GoToPointAction

# goal
float32 x
float32 y
float32 theta
---
# result
bool ok               # true only if the request isn't cancelled
---
# feedback
int32 status
float32 actualX
float32 actualY
float32 actualTheta
float32 distance      # distance from the goal
float32 progress      # ( 1 - actual_distance / distance_from_start )
float32 progressTime  # time from the request
